### DISCUSSION TITLE
Installation help required

### DISCUSSION THREAD

**Discussion by parthrmistri** [jens-maus/thinRoot#19] [Q&A] [answered]
Hi, Could you help to create image for raspberry pi3?

I have tried below steps but didn't help.

**Comment by jens-maus** [comment, marked as answer]
Sorry for the late answer, but please note that we do create ready-usable images for the RaspberryPi3 now. In addition, the build for rpi3 should succeed nicely now and should allow to create your own images. Furthermore, we also spent some time in doing some basic documentation in the top-level README of this repo and also added some demo tftpboot directory in the repository as well (see https://github.com/jens-maus/thinRoot/tree/master/tftpboot).

**Reply by parthrmistri** [reply]
Hi Jens, Thanks for reply, could you please help with building customized image and also share ready available ISO which i can directly boot via USB Drive into PC?

**Reply by jens-maus** [reply]
Sure. Just have a look at the latest releases page (https://github.com/jens-maus/thinRoot/releases/latest) and have a look for the "Assets" section there. There you would find precompiled images for different platforms including a generic-x86_64 image that can be directly booted via PXE/TFTP. Furthermore, please have a look at the main README of the repository where I added some more additional information on how to build your own customized image. Apart from that you would of course have to walk on your own ;-)
